packages/tailwind/src/kz-spacing.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { filterOutNonPixelTokens } from "./kz-spacing"
+
+const numericKeyPairs = {
+  0: "0",
+  1: ".0625rem",
+  2: ".125rem",
+}
+const nonNumericKeyPairs = {
+  xs: "0.375rem",
+  sm: "0.75rem",
+}
+
+const dummyData = {
+  ...numericKeyPairs,
+  ...nonNumericKeyPairs,
+}
+
+describe("filterOutNonPixelTokens", () => {
+  const result = filterOutNonPixelTokens(dummyData)
+
+  it("It removes pairs with non-numeric keys", () => {
+    Object.keys(nonNumericKeyPairs).forEach(key => {
+      expect(result).not.toHaveProperty(key)
+    })
+  })
+  it("Keeps all numeric keys", () => {
+    expect(result).toEqual(numericKeyPairs)
```

Comment:
```suggestion
const result = filterOutNonPixelTokens({
	0: "0",
	xs: "0.375rem",
	1: ".0625rem",
	sm: "0.75rem",
	2: ".125rem",
})

expect(result).toEqual({
	0: "0",
	1: ".0625rem",
	2: ".125rem",
})
```

I think it's actually a little easier to read without too many variables in a test, especially since there aren't many in this test. Something like this is very explicit and clear what we're expecting to get.

Reply:
Alrighty 👍